app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -271,10 +274,17 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
             className="flex flex-col items-center w-full pt-8"
             data-testid="weekly-picks"
           >
-            <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
-              Week {week} pick
-            </h1>
-
+            <Heading 
```

Comment:
Why not test that H1 is on the screen and has the correct value as well?

Reply:
@ryandotfurrer How come all headings do not have `pb-8` by default? Heading's usually have margin from the next element.